### DISCUSSION TITLE
Can't get @apply working in scss file, browser states unknown property

### DISCUSSION THREAD

**Discussion by iangregsondev** [tailwindlabs/tailwindcss#3121] [Help] [answered]
Hi,

I am trying to create my own class that inherits some of tailwind's classes using @apply 

```
.testme {
  @apply px-4 py-2 rounded font-semibold bg-gray-200 text-black
  background-color: red;
}
```

and

```
    <div className="testme">
        THIS IS STYLED WITH MY OWN CLASS, THAT USES @APPLY from TAILWIN
    </div>
```

When i view it in the browser it reports unknown property

![image](https://user-images.githubusercontent.com/2370364/102481108-90c88d80-4061-11eb-9f3f-44bc39b18175.png)

I am using postcss-import like so

```
module.exports = {
  plugins: [require("postcss-import"), require("tailwindcss"), require("autoprefixer")],
}
```

I have published a small repo that outlines the issue (run "gatsby develop" ) (The files are in pages, under index.tsx and index.css)

It's just a sample repo with most of my other stuff removed so to focus on the issue..

https://github.com/iangregsondev/tailwind-apply-issue

As far as I can see I have followed the documentation but I can't get it to work 

Anybody know what's wrong ?

Thanks in advance

**Comment by iangregsondev** [comment]
I have also updated the repo to include the @imports and no change

```
@import 'tailwindcss/utilities';
@import 'tailwindcss/base';
@import 'tailwindcss/components';

.testme {
  @apply px-4 py-2 rounded font-semibold bg-gray-200 text-black
  background-color: red;
}
```

**Comment by simonswiss** [comment, marked as answer]
Hello!

In your custom CSS, you need to add a semicolon `;` at the end of the `@apply` line, since you have another declaration after that (`background-color: red;`)

```diff
.testme {
-  @apply px-4 py-2 rounded font-semibold bg-gray-200 text-black
+  @apply px-4 py-2 rounded font-semibold bg-gray-200 text-black;
  background-color: red;
}
```

I tried this on your repo, and this solved it 👍

**Reply by iangregsondev** [reply]
Hi Simon, thanks, Yes I added the missing semicolon yesterday but I still couldn't get it to work..

I will re-check out my own repo and try again :-)

I did manage to get to the bottom of the issues. So it works with CSS files but scss files it did not..

In the end, as my scss file is stored elsewhere, I added this to the plugin in gatsby

```
    {
      resolve: `gatsby-plugin-sass`,
      options: {
        includePaths: ["./node_modules"],
        postCssPlugins: [require("tailwindcss"), require("autoprefixer")],
        // precision: 8, // SASS default: 5
      },
    },
```

I think this might be the solution but then I got errors about using v8 of PostCSS - i think gatsby is still on an old version of PostCSS.

I think found a work around on tailwindcss about backwards compatibility - tried it - but it doesn't work.

I will look again today


Thanks.

**Comment by khtodod** [comment]
I added the missing semicolon but I still couldn't get it to work.
